symengine/python/cmake/cython_test.pyx

```diff
@@ -83,3 +83,8 @@ cdef extern from "<symengine/add.h>" namespace "SymEngine":
 
     cdef cppclass Add(Basic):
         void as_two_terms(const Ptr[RCP[Basic]] &a, const Ptr[RCP[Basic]] &b)
+
+
+# Test that cpdef void is supported (https://github.com/cython/cython/commit/7df9d654b3e6c3d5812a95904872d35a3c94ef77)
```

Comment:
Thanks. This would be really cool if we can do that. I think it's safe to assume that the Cython version that is available on the system is 2 or 3 years old, which is still a very decent version. I would only use newer stuff if we absolutely have to.